From 49a87e4cbb898ab3295e3d6b62929a906b41b24f Mon Sep 17 00:00:00 2001
From: Robert Marko <robimarko@gmail.com>
Date: Tue, 11 May 2021 13:29:33 +0200
Subject: [PATCH 1006/1018] arm64: dts: add OpenWrt DTS files

Lets add custom OpenWrt DTS files to Makefile.

Signed-off-by: Robert Marko <robimarko@gmail.com>
[mrnuke: Replace ipq807x dtbs with ipx601x]
Signed-off-by: Alexandru Gagniuc <mr.nuke.me@gmail.com>
---
 arch/arm64/boot/dts/qcom/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/qcom/Makefile b/arch/arm64/boot/dts/qcom/Makefile
index 70516508be56..b9d3b1d94e76 100644
--- a/arch/arm64/boot/dts/qcom/Makefile
+++ b/arch/arm64/boot/dts/qcom/Makefile
@@ -4,6 +4,8 @@ dtb-$(CONFIG_ARCH_QCOM)	+= apq8094-sony-xperia-kitakami-karin_windy.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= apq8096-db820c.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= apq8096-ifc6640.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq6018-cp01-c1.dtb
+dtb-$(CONFIG_ARCH_QCOM)	+= ipq6018-eap610-outdoor.dtb
+dtb-$(CONFIG_ARCH_QCOM)	+= ipq6018-mr7350.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq8074-hk01.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq8074-hk10-c1.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq8074-hk10-c2.dtb
-- 
2.37.1

